# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-05-14 16:15+0800\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: account/views.py:75
msgid "Username and password cannot be empty"
msgstr "旧密码不能为空"

#: account/views.py:86
msgid "User does not exist.Please register an account first"
msgstr "帐号不存在，请先注册帐号"

#: account/views.py:107 account/views.py:535
msgid "Account is still login in other platform. Kick the user?"
msgstr "此帐号在其他地方还保持登入，是否踢掉连线"

#: account/views.py:114 account/views.py:542
msgid "Username or password is not correct"
msgstr "帐号或密码不正确"

#: account/views.py:126 account/views.py:558
msgid "You have successfully logged out"
msgstr "已成功登出"

#: account/views.py:158 account/views.py:380
msgid "Invalid email address"
msgstr "信箱格式不正確"

#: account/views.py:183
msgid ""
"User has successfully created. Please check email for account validation"
msgstr "已经成功建立使用者，请进行信箱验证"

#: account/views.py:188
msgid ""
"User has successfully created. Email sent failed. Please resend the email to "
"validate the account"
msgstr "已经成功建立使用者，信箱传送失败，请再次传送验证信"

#: account/views.py:193
msgid "User name has existed"
msgstr "使用者名称已存在"

#: account/views.py:212
msgid "Please register an account first"
msgstr "请先注册帐号"

#: account/views.py:215 account/views.py:278
msgid "Account has been confirmed. No need to get confirmation email again"
msgstr "帐号已验证，不需再次传送验证信"

#: account/views.py:227
msgid "Please wait for a while to resend the code"
msgstr "请稍候再尝试"

#: account/views.py:245
msgid "Confirmation email has been sent"
msgstr "已寄送验证信"

#: account/views.py:247
msgid "Failed to send email"
msgstr "传送信件失败"

#: account/views.py:272
msgid "Invalid code"
msgstr "认证失败"

#: account/views.py:285
msgid "User name has been changed"
msgstr "帐号已更改"

#: account/views.py:289
msgid "Account has been validated"
msgstr "帐号已验证"

#: account/views.py:292
msgid "Account validation failed"
msgstr "帐号验证失败"

#: account/views.py:310
msgid "Send reset email failed. Please send again"
msgstr "传送信件失败，请在传送一次"

#: account/views.py:313
msgid ""
"New password has sent to your mailbox. Please check and use the new password "
"to login"
msgstr "新的密码已寄送至信箱，请收信后使用新密码来登入"

#: account/views.py:317
msgid "User does not exist"
msgstr "使用者不存在"

#: account/views.py:368 account/views.py:447 account/views.py:475
#: account/views.py:603 account/views.py:648 account/views.py:686
#: account/views.py:735 chatbot/views.py:182 chatbot/views.py:210
#: chatbot/views.py:240 chatbot/views.py:291 chatbot/views.py:304
#: chatbot/views.py:323 chatbot/views.py:333 chatbot/views.py:380
#: chatbot/views.py:393 chatbot/views.py:412 chatbot/views.py:422
#: chatbot/views.py:466 chatbot/views.py:470 chatbot/views.py:492
#: chatbot/views.py:496 chatbot/views.py:500 chatbot/views.py:552
#: chatbot/views.py:556 chatbot/views.py:578 chatbot/views.py:582
#: chatbot/views.py:587 faq/views.py:109 faq/views.py:181 faq/views.py:193
#: faq/views.py:216 faq/views.py:226 faq/views.py:286 faq/views.py:298
#: faq/views.py:323 faq/views.py:333 faq/views.py:476 paidtype/views.py:103
#: report/views.py:88 taskbot/views.py:88 taskbot/views.py:123
#: taskbot/views.py:153
msgid "Not found"
msgstr "资料不存在"

#: account/views.py:384
msgid "Username has existed.Please use other one"
msgstr "帐号名称已存在，请输入其他名称"

#: account/views.py:392
msgid "Sending confirmation email failed"
msgstr "寄送认证信失败"

#: account/views.py:401
msgid "Confirmation email has sent.Please check your mailbox and login again"
msgstr "认证信已寄送，请至信箱进行认证后登入"

#: account/views.py:410
msgid "Old password cannot be empty"
msgstr "旧密码不能为空"

#: account/views.py:416 account/views.py:616
msgid "Old password is not correct"
msgstr "旧密码不正确"

#: account/views.py:422 account/views.py:614
msgid "Password has updated.Please login again"
msgstr "密码已更新，请重新登入"

#: account/views.py:436
msgid "Please select one language"
msgstr "请选择语言"

#: account/views.py:439 account/views.py:640 chatbot/views.py:196
#: chatbot/views.py:339 chatbot/views.py:428 chatbot/views.py:510
#: chatbot/views.py:597 faq/views.py:230 faq/views.py:338 paidtype/views.py:121
#: taskbot/views.py:110
msgid "Update succeeded"
msgstr "更新成功"

#: account/views.py:442 account/views.py:491 account/views.py:596
#: account/views.py:702 account/views.py:775 chatbot/views.py:171
#: chatbot/views.py:198 chatbot/views.py:253 chatbot/views.py:341
#: chatbot/views.py:430 chatbot/views.py:485 chatbot/views.py:516
#: chatbot/views.py:571 chatbot/views.py:603 faq/views.py:207 faq/views.py:232
#: faq/views.py:313 faq/views.py:340 faq/views.py:420 paidtype/views.py:88
#: paidtype/views.py:123 taskbot/views.py:79 taskbot/views.py:112
#: taskbot/views.py:166
msgid "No content"
msgstr "无内容"

#: account/views.py:452 account/views.py:653 chatbot/views.py:213
#: taskbot/views.py:126
msgid "Please confirm the deletion first"
msgstr "请先确认是否删除"

#: account/views.py:459 account/views.py:660
msgid "Deleting account failed"
msgstr "删除帐号失败"

#: account/views.py:481 account/views.py:692 chatbot/views.py:244
#: taskbot/views.py:157
msgid "Please enter the password"
msgstr "请输入密码"

#: account/views.py:487 account/views.py:698 chatbot/views.py:249
#: taskbot/views.py:162
msgid "Delete confirmed"
msgstr "确认删除"

#: account/views.py:489 account/views.py:700 chatbot/views.py:251
#: taskbot/views.py:164
msgid "Password is not correct"
msgstr "密码不正确"

#: account/views.py:512
msgid "User does not exist. Please contact admin."
msgstr "帐号不存在，请联系客服人员"

#: account/views.py:516
msgid "Only staff can login this page"
msgstr "必须为管理员帐号"

#: account/views.py:592
msgid "Create new agent succeeded"
msgstr "创建管理员帐号成功"

#: account/views.py:594 faq/views.py:205 faq/views.py:311 history/views.py:145
msgid "Create failed"
msgstr "创建失败"

#: account/views.py:624
msgid "User name existed"
msgstr "使用者名称已存在"

#: account/views.py:634
msgid "Do not support this language"
msgstr "不支援此语言"

#: account/views.py:645 account/views.py:682
msgid "Superuser cannot be deleted"
msgstr "不能删除超级管理员帐号"

#: account/views.py:749
msgid "Invalid type"
msgstr "方案不正确"

#: account/views.py:771
msgid "Update successed"
msgstr "更新成功"

#: account/views.py:773
msgid "paid_type cannot be empty"
msgstr "方案不能为空"

#: chatbot/views.py:119
msgid "Reach bot create limitation"
msgstr "已到建立机器人数量上限"

#: chatbot/views.py:131
msgid "Reach faq create limitation"
msgstr "已到问答数量上限"

#: chatbot/views.py:162 taskbot/views.py:70
msgid "Create bot failed. Cause by NLU error."
msgstr "建立机器人失败"

#: chatbot/views.py:169 taskbot/views.py:77
msgid "Create bot failed"
msgstr "建立机器人失败"

#: chatbot/views.py:221 taskbot/views.py:134
msgid "Deleting bot failed"
msgstr "删除机器人失败"

#: chatbot/views.py:295 chatbot/views.py:308 chatbot/views.py:327
msgid "Not allowed to get user line data"
msgstr "不允许此操作"

#: chatbot/views.py:384 chatbot/views.py:397 chatbot/views.py:416
msgid "Not allowed to get user fb data"
msgstr "不允许此操作"

#: faq/views.py:119
msgid "Faq group exceeded upper limit"
msgstr "问答组已到数量上限"

#: faq/views.py:128
msgid "Create faq group failed"
msgstr "创建问答组失败"

#: faq/views.py:369 faq/views.py:381
msgid "File type is not correct. Should be type utf8 or big5."
msgstr "格式不正确，需为utf8或big5"

#: faq/views.py:377
msgid "CSV column is not correct"
msgstr "CSV 栏位数量不正确"

#: faq/views.py:406
msgid "Group over limitation"
msgstr "问答组超过上限"

#: faq/views.py:415
msgid "Upload succeeded"
msgstr "上传成功"

#: faq/views.py:418 faq/views.py:461
msgid "Bot not found"
msgstr "机器人不存在"

#: faq/views.py:482
msgid "Training bot succeeded"
msgstr "成功训练机器人"

#: history/views.py:117
msgid "Chatbot not found"
msgstr "机器人不存在"

#: history/views.py:124
msgid "Matching result not found"
msgstr "推荐纪录不存在"

#: history/views.py:131
msgid "FAQ group does not exisit"
msgstr "问答组不存在"

#: history/views.py:142
msgid "Create new question successed"
msgstr "成功建立新问题"

#: paidtype/views.py:65 paidtype/views.py:98
msgid "Not allowed"
msgstr "不允许此操作"

#: paidtype/views.py:80
msgid "Create paidtype failed"
msgstr "建立方案失败"

#: taskbot/views.py:97
msgid "Key missing or empty: robot_name"
msgstr "缺少robot_name"

#~ msgid "Something went wrong. Please try again"
#~ msgstr "错误，请再尝试一次"

#~ msgid "English"
#~ msgstr "英文"

#~ msgid "Simplify Chineses"
#~ msgstr "简体中文"

#~ msgid "Traditional Chineses"
#~ msgstr "繁体中文"
